### PR TITLE
Final Touch project complete

### PR DESCRIPTION
**Pull request by baqar-abbas**
Add Back ground colors and styling like hover on button, transitions effects and animation effects for styling.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -35,6 +45,15 @@ h1 {
...
   margin-bottom: 2rem;
+  background-color: #d6cff0;
+  padding: 20px 10px;
+}
+
+.recentScore h2 {
+  color: black;
+  animation-name: example;
+  animation-duration: 0.7s;
+  animation-iteration-count: infinite;
```
Comment (Shedrack-Sunday) [inline src/index.css:58]: - [ ] Nice work you did here 👏 . However, Kindly Move the `Leaderboard` heading to the left as it's aligned on the wireframe.  This gives a better UX and user view experience in the aspect of how well the contents are placed 👍 

## Wireframe:
image

## Your view:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @baqar-abbas      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked
✅ Proper PR summary 
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, You need to make a little adjustment on the placing of the `h1` on your design in reference to the wireframe, and also kindly fix the broken `MIT` Link on your README

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.css (hunk 1 above)
Comment (baqar-abbas) [reply src/index.css:58]: Suggested change has been implemented. Screen shot attached below for your reference.
image

Comment (baqar-abbas) [reply src/index.css:58]: Broken link to the License file has also been fixed. Screen shot attached below for your reference.
image
image

**Comment by baqar-abbas** [comment]
Suggested changes have been incorporated. Submitting it for a review again.

**Review by BertrandConxy — APPROVED** [review]
Hi @baqar-abbas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**